apps/babanews/pages/[slug].tsx

```diff
@@ -52,6 +52,6 @@ export const getStaticProps: GetStaticProps = async ({params}) => {
 
   return {
     props,
-    revalidate: 60 // every 60 seconds
+    revalidate: !page.data?.page ? 1 : 60 // every 60 seconds
```

Comment:
This makes sure that pages that are in draft will be invalidated basically immediately. Makes publishing faster.